pkg/config/config.go

```diff
@@ -232,6 +232,11 @@ func (env *TestEnvironment) doAutodiscover() {
 		env.ContainersToExcludeFromConnectivityTests[cid] = ""
 	}
 
+	// Delete all remaining sessions before re-creating them
```

Comment:
Move it up to somewhere around line 206?

Reply:
fixed